README.md

```diff
@@ -90,11 +90,11 @@ NOTE: Containers will take a little while to start their work. You can see the s
 
 ### Docker Compose
 
-Testing on docker-compose is relatively less resource and time consuming.
+Installing on docker-compose is relatively less resource and time consuming.
 
 1. Install docker-compose: `pip install docker-compose`
 
-2. Change the following options according to your system: `DJANGO_SECRET_KEY`, `DB_USER`, `DB_PASS`, `DJANGO_DEF_EMAIL`, `DASHBOARD_DOMAIN`, `CONTROLLER_DOMAIN`, `RADIUS_DOMAIN`, `TOPOLOGY_DOMAIN`. Optionally, you may change any other setting as well. A detailed document of the available variables can be found [here](docs/ENV.md).
+2. Change the following options in `.env` file according to your system: `DJANGO_SECRET_KEY`, `DB_USER`, `DB_PASS`, `DJANGO_DEF_EMAIL`, `DASHBOARD_DOMAIN`, `CONTROLLER_DOMAIN`, `RADIUS_DOMAIN`, `TOPOLOGY_DOMAIN`. Optionally, you may change any other setting as well. A detailed document of the available variables can be found [here](docs/ENV.md).
```

Comment:
Can you please change `DJANGO_DEF_EMAIL` -> `EMAIL_DJANGO_DEFAULT`?
This one is my mistake, must have not noticed while typing this docs! :smile:

Reply:
Ok :smile: